### ISSUE TITLE
cant download

### ISSUE THREAD

**Issue by 0x77x** [braycarlson/alphares#19]

**Comment by braycarlson** [comment]
I will need more information. What browser are you using? Is it blocked by Chrome or Firefox? Is it blocked by anti-virus?

**Comment by braycarlson** [comment]
I am going to close this issue due to inactivity and because I have created a FAQ to address some of these common issues. You can read it here: https://github.com/braycarlson/alphares#faq. If you are still experiencing issues after reading the FAQ, then feel free to create a new issue.
